DatadogInternal/Sources/Models/SessionReplay/SessionReplayConfiguration.swift

```diff
@@ -57,6 +57,15 @@ public enum TouchPrivacyLevel: String {
     case hide
 }
 
+/// Privacy level for overriding global privacy settings in Session Replay.
+public enum HiddenPrivacyLevel: String {
```

Comment:
/question

is this something customer can set or get? If yes, we must forward pass through SR package. We don't want customers to force import internal.

Reply:
Yes, customers can set it. I placed it here along the other privacy levels. If we move this enum to the SR module, then we should move the others as well imo.

Reply:
You would need to do something like

https://github.com/DataDog/dd-sdk-ios/blob/783407bc8ea9d5027e874b59212116993a8bea2a/DatadogTrace/Sources/TraceConfiguration.swift#L22

if you keep them in internal, but moving to SR module makes more sense if these types are not shared in other modules.

not blocking as we have other similar things already, but we must address this before GA.